PostHog.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "PostHog"
-  s.version          = "3.24.2"
+  s.version          = "3.25.0"
```

Comment:
versioning is done by the release process https://github.com/PostHog/posthog-ios/blob/main/RELEASING.md

Reply:
got it, will amend